### PR TITLE
changed month abbreviations in month_plot pt2

### PR DESCRIPTION
**Pull request by oswaldochan**
to use three letter abbreviations (i.e. 'Jan', 'Feb') instead a single letter abbreviation (i.e. 'j', 'f')

sorry I'm not completely sure on how to use github, I just made a small change on how months are displayed so it looks prettier

- [ ] closes #xxxx
- [x] tests added / passed. 
- [x] code/documentation is well formatted.  
- [x] properly formatted commit message. See 
      [NumPy's guide](https://docs.scipy.org/doc/numpy-1.15.1/dev/gitwash/development_workflow.html#writing-the-commit-message). 

<details>

**Notes**:

* It is essential that you add a test when making code changes. Tests are not 
  needed for doc changes.
* When adding a new function, test values should usually be verified in another package (e.g., R/SAS/Stata).
* When fixing a bug, you must add a test that would produce the bug in master and
  then show that it is fixed with the new code.
* New code additions must be well formatted. Changes should pass flake8. If on Linux or OSX, you can
  verify you changes are well formatted by running 
  ```
  git diff upstream/master -u -- "*.py" | flake8 --diff --isolated
  ```
  assuming `flake8` is installed. This command is also available on Windows 
  using the Windows System for Linux once `flake8` is installed in the 
  local Linux environment. While passing this test is not required, it is good practice and it help 
  improve code quality in `statsmodels`.
* Docstring additions must render correctly, including escapes and LaTeX.

</details>

### CODE REVIEW

File: statsmodels/graphics/tsaplots.py [hunk 1]
```diff
@@ -407,7 +407,8 @@ def month_plot(x, dates=None, ylabel=None, ax=None):
     else:
         x = pd.Series(x, index=pd.PeriodIndex(dates, freq="M"))
 
-    xticklabels = ["j", "f", "m", "a", "m", "j", "j", "a", "s", "o", "n", "d"]
+    xticklabels = ["Jan", "Feb", "Mar", "Apr", "May", "Jun",
```
Comment (bashtage) [inline statsmodels/graphics/tsaplots.py:410]: Should we localize these.  Could use

```
import datetime as dt

year = dt.datetime.now().year
xticklabels  = [dt.datetime(year, i+1, 1).strftime("%b") for i in range(12)]
```

**Comment by jseabold** [comment]
@bashtage @oswaldochan Hope you don't mind, but I went ahead and updated your PR with the review suggestion.

I forgot to rebase out my editor's isort commit. Can take it out, if it's too noisy. Will turn it off in the future.

**Comment by bashtage** [comment]
Thanks @jseabold . There was one other small issue which is in #7409.

**Comment by bashtage** [comment]
Thanks @oswaldochan  and @jseabold - Your contributions were merged in #7409.

**Pull request closed without merging**